client/templates/juror-management/attendance/confirmed/confirmed.njk

```diff
@@ -92,10 +92,10 @@
     <div class="govuk-grid-column-full govuk-body">
 
       <b>
-        <span id="absentJurors">{{ absentJurors.length }}</span> jurors confirmed as absent
+        <span id="absentJurors">{{ absentJurors.length }}</span> juror{% if listedJurors.length > 1 %}s{% endif %} confirmed as absent
       </b>
 
-      <table class="govuk-table {% if listedJurors.length === 0 %}js-hidden{% endif %}"
+      <table class="govuk-table {% if absentJurors.length === 0 %}js-hidden{% endif %}"
```

Comment:
should these be both `absentJurors.length`?

Reply:
Thank you, fixed now. I was un-doing some other changes and must've un-done one too many times 😅